common/src/backends/tracing.rs

```diff
@@ -34,6 +34,7 @@ use crate::tracing::{
 };
 
 const OTLP_ADDRESS: &str = "http://otel-collector:4317";
+const LOGGER_URI: &str = "http://127.0.0.1:8080";
```

Comment:
Maybe this should be `http://logger:8000`?

Reply:
Yes, that makes sense.